src/components/Datatable/Table/Table.tsx

```diff
@@ -139,6 +140,7 @@ TableProps<D>): React.ReactElement => {
       initialState: {
         // PAGINATION
         pageSize: defaultPageSize,
+        pageIndex: defaultPageIndex || 0,
```

Comment:
Maybe would be better to set the default value in https://github.com/securityscorecard/design-system/blob/alpha/src/components/Datatable/defaultConfigs.ts as every other default